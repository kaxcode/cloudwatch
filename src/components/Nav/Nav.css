.Nav__Img {
  width: 100px;
  margin-top: 20px;
}
