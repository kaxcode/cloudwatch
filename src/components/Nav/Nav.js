import React from 'react';
import { Link } from 'react-router-dom';
import NavCloud from '../../assets/blank_cloud.svg';

const Nav = () => {
  return (
    <Link to="/">
      <img className="nav" src={NavCloud} alt="Click to return home" />
    </Link>
  );
};

export default Nav;
