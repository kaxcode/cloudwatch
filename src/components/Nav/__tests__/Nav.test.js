import React from 'react';
import Enzyme from 'enzyme';
import Nav from '../Nav';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Stopwatch', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav />);
  });

  it('renders correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
