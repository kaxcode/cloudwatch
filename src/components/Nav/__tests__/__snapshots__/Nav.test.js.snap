// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stopwatch renders correctly 1`] = `
<Link
  replace={false}
  to="/"
>
  <img
    alt="Click to return home"
    className="nav"
    src="blank_cloud.svg"
  />
</Link>
`;
